### ISSUE TITLE
OpenShift 4.3 new default: managementState: Removed for image registry

### ISSUE THREAD

**Issue by rbo** [RedHat-EMEA-SSA-Team/hetzner-ocp4#79]
```shell
$ oc get configs.imageregistry.operator.openshift.io/cluster -o yaml | grep managementState
  managementState: Removed
```

https://docs.openshift.com/container-platform/4.3/release_notes/ocp-4-3-release-notes.html#ocp-4-3-image-registry-operator

Means, new cluster installation don't have a running registry.

**Comment by rbo** [comment]
Work-a-round / solution:
```
[root@runner hetzner-ocp4]# oc get configs.imageregistry.operator.openshift.io/cluster -o yaml | grep -B5 managementState
spec:
  defaultRoute: false
  disableRedirect: false
  httpSecret: 1f9b452cad839aeba43095e640d7cac29fcb78ec5c221c88f60541d651ad116fe6528247e88e85a9aea49fbfe8ab04bfdd257ce0e5b85b46d5394e13093c72c5
  logging: 2
  managementState: Removed
[root@runner hetzner-ocp4]# oc patch configs.imageregistry.operator.openshift.io/cluster --type merge --patch '{\"spec\":{\"managementState\": \"Managed\"}}'
Error from server: Invalid JSON Patch
[root@runner hetzner-ocp4]# oc get configs.imageregistry.operator.openshift.io/cluster -o yaml | grep -B5 managementState
spec:
  defaultRoute: false
  disableRedirect: false
  httpSecret: 1f9b452cad839aeba43095e640d7cac29fcb78ec5c221c88f60541d651ad116fe6528247e88e85a9aea49fbfe8ab04bfdd257ce0e5b85b46d5394e13093c72c5
  logging: 2
  managementState: Managed
```

**Comment by rbo** [comment]
Solved, created PR #81
